.travis.yml

```diff
@@ -60,6 +60,7 @@ env:
     #  'TESTCASE=pythran/tests/test_numpy_func1.py',
     #  'TESTCASE=pythran/tests/test_numpy_func2.py',
     #  'TESTCASE=pythran/tests/test_numpy_func3.py',
+    #  'TESTCASE=pythran/tests/test_numpy_random.py',
```

Comment:
I'm pretty sure it is, I did that a couple of days ago. I though you would get a mail :/

So unless I forgot sthg (again :/) it should be good to merge.